### ISSUE TITLE
希望可以支持parcel

### ISSUE THREAD

**Issue by gxy5202** [antfu/unplugin-vue-components#500]
### Clear and concise description of the problem

parcel现在的打包速度很快，能否支持parcel的按需加载？

### Suggested solution

适配parcel的插件系统

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Lazerbeak12345** [comment]
I agree - since parcel is so fast it'd be really nice to support this.

I noticed that the Vue CLI guide in the [installation instructions][inst] is pretty much the same as the Webpack guide, so I wonder if a short-term workaround would work doing something similar.

[Here's more information about Parcel v2][pv2], which came out in 2021, and as part of its "new" features it has

> 🚀 Massive performance improvements, including a new JavaScript compiler written in Rust, and a more parallelized architecture that takes advantage of all of your CPU cores.

I've noticed a _lot_ of outdated information around parcel, where people haven't even heard of v2, and assume it's still slow, but I've found it to be comparable speed to vite, with less config needed, generally.

[inst]: https://github.com/unplugin/unplugin-vue-components#installation
[pv2]: https://parceljs.org/blog/v2/

**Comment by sxzz** [comment]
Track upstream issue: https://github.com/unjs/unplugin/issues/30
